### DISCUSSION TITLE
ダークモード時はTwitterのウィジェットもダークテーマにしてほしい

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#436] [要望・アイデア・不具合]
Qiitaでは、Twitterのツイートを埋め込むことができます。

- [Qiitaで埋め込み可能なコンテンツ一覧 - Qiita](https://qiita.com/Qiita/items/612e2e149b9f9451c144)

ダークモード時にウィジェットがデフォルトのテーマで表示されるとコントラストがきついので、ダークモード時にはダークテーマで表示されるようにしてほしいです。

## 表示例

通常：

<img width="842" alt="スクリーンショット 2023-02-20 16 25 18" src="https://user-images.githubusercontent.com/20086673/220039406-1cecd5a8-5b4c-4b23-87fb-03e35dcc6dba.png">

ダークテーマ：

<img width="842" alt="スクリーンショット 2023-02-20 16 25 22" src="https://user-images.githubusercontent.com/20086673/220039413-aa410e19-2ed5-4e35-9b15-518dce7a3e2d.png">

**Comment by tanaka-asami415** [comment]
@munierujp 
ありがとうございます🙏検討いたします！

**Comment by wataru86** [comment]
@munierujp
ご報告が遅くなり申し訳ありません 🙇 
ダークモード時にツイート埋め込みもダークテーマになるように変更いたしました ✨
